Mage/src/main/java/mage/constants/SubType.java

```diff
@@ -99,6 +100,7 @@
     DEMON("Demon", SubTypeSet.CreatureType, false),
     DESERTER("Deserter", SubTypeSet.CreatureType, false),
     DEVIL("Devil", SubTypeSet.CreatureType, false),
+    DINOSAUR("Dinosaur", SubTypeSet.CreatureType, false),
```

Comment:
Until the release of Ixilan, Dinosaur is a Fantasy Type.  Please be sure to mark it as such.